### ISSUE TITLE
coolpc.com.tw & coolpc.tw :ads

### ISSUE THREAD

**Issue by weijen6** [uBlockOrigin/uAssets#15883]
### Prerequisites

- [X] This is not a [support issue or a question](https://old.reddit.com/r/uBlockOrigin/)
- [X] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uAssets/issues) to avoid opening a duplicate issue
- [X] I am running the latest version of uBlock Origin

### I tried to reproduce the issue when...

- [X] uBO is the only extension
- [X] uBO with default lists/settings
- [X] using a new, unmodified browser profile

### URL(s) where the issue occurs

'https://coolpc.tw/evaluate.php'
'https://coolpc.com.tw/evaluate.php'

### Describe the issue

After turning on ublock to block ads, the page becomes gray and cannot click anything

### Screenshot(s)

![chrome_TR1cBiHTzg](https://user-images.githubusercontent.com/32688959/205277546-d5a73d82-9e0b-454a-a369-fd5835ecb21c.png)

### uBlock Origin version

uBlock Origin 1.45.2

### Browser name and version

Chrome 107.0.5304.12

### Settings

none

**Comment by mapx-** [comment]
I guess you are using some annoyances list, add:

```
coolpc.tw#@##mycookie
coolpc.com.tw#@##mycookie
```

**Comment by krystian3w** [comment]
Fixed in AdGuard.

coolpc.com.tw need future investigation:

https://github.com/AdguardTeam/AdguardFilters/blob/79f20f1eec0821ab25e42be436a9e9101c6fdff9/AnnoyancesFilter/Cookies/sections/cookies_allowlist.txt#L2487-L2488 - old https://github.com/AdguardTeam/AdguardFilters/commit/1504fe660270b5f900a50558b56573389841c4c5

Also AdamWr idea is slow, uBo shows pop-up ~3 seconds:

```adb
coolpc.com.tw,coolpc.tw##+js(nano-setTimeout-booster, SkipMsg(), *, 0.02)
```

**Comment by candela97** [comment]
Yeah that `#mycookie` filter from Fanboy's annoyance breaks the site as I mentioned before. How did it make it to  AdGuard annoyances 😅.

The new filter kinda works, but it's no better than manually clicking outside the popup imo.
